apps/projects/app/components/Content/IssueDetail/DetailsCard.js

```diff
@@ -42,69 +45,46 @@ const DetailsCard = ({ issue }) => {
-    <Text size="large" css="display: flex; align-items: center">
-      <img src={author.avatarUrl} alt="user avatar" css="margin: 8px; width: 24px; border-radius: 50%;" />
-      <Link
-        href={author.url}
-        target="_blank"
-        style={{ textDecoration: 'none', color: `${theme.link}` }}
-      >
-        {author.login}
-      </Link>
-    </Text>
-  )
-  OpenedBy.propTypes = userGitHubShape
-
   const SummaryTable = ({ issue }) => (
-    <div css={`
-      display: grid;
-      grid-template-columns: repeat(${issue.hasBounty ? '3' : '4'}, 1fr);
-      grid-template-rows: auto;
-      ${issue.hasBounty ? `
+    <div css={
+      `${issue.hasBounty && `
+        display: grid;
+        grid-template-rows: auto;
+        grid-gap: ${3 * GU}px;
+        grid-template-columns: 1fr 1fr;
         grid-template-areas:
-          'deadline exp openedby' 'description description description'
-        ` : `
-        grid-template-areas:
-          'description description description openedby'
-        `
-    };
-      grid-gap: 12px;
-      align-items: stretch;
-    `}>
+          'deadline exp'
+          'description description'
+        ;`}
+        margin-top: ${1.5 * GU}px;
+      `
+    }>
       {issue.hasBounty ? (
         <React.Fragment>
-          <SummaryCell label="Time until due" grid="deadline">
-            <Timer end={new Date(issue.deadline)} />
+          <SummaryCell label="Deadline" grid="deadline">
+            <Text.Block>
+              <DeadlineDistance date={issue.deadline} />
+            </Text.Block>
           </SummaryCell>
           <SummaryCell label="Difficulty" grid="exp">
             <Text.Block>
               {issue.expLevel}
             </Text.Block>
           </SummaryCell>
-          <SummaryCell label="Opened by" grid="openedby">
-            <OpenedBy author={issue.author} />
-          </SummaryCell>
           <SummaryCell label="Description" grid="description">
             <Markdown
               content={issue.body || 'No description available'}
-              style={{ marginTop: '20px', marginBottom: '20px' }}
+              style={{ marginBottom: (2 * 16) + 'px' }}
             />
           </SummaryCell>
         </React.Fragment>
       ) : (
-        <React.Fragment>
-          <SummaryCell label="Description" grid="description">
-            <Markdown
-              content={issue.body || 'No description available'}
-              style={{ margin: (2 * GU) + 'px 0', width: '100%' }}
-            />
-          </SummaryCell>
-          <div css="grid-area: openedby">
-            <Label text="Opened by" />
-            <OpenedBy author={issue.author} />
-          </div>
-        </React.Fragment>
+        <SummaryCell label="Description" grid="description">
+          <Markdown
+            content={issue.body || 'No description available'}
+            style={{ marginBottom: (2 * GU) + 'px', width: '100%' }}
+          />
+        </SummaryCell>
```

Comment:
* if has bounty
  * render deadline
  * render difficulty
  * render description
* else
  * render description

Since we're always rendering description, can we move rendering description outside of the ternary?

```js
{issue.hasBounty &&
  <>
    <SummaryCell label="Deadline" grid="deadline">
      <Text.Block>
        <DeadlineDistance date={issue.deadline} />
      </Text.Block>
    </SummaryCell>
    <SummaryCell label="Difficulty" grid="exp">
      <Text.Block>
        {issue.expLevel}
      </Text.Block>
    </SummaryCell>
  </>
}
<SummaryCell label="Description" grid="description">
  ...
</SummaryCell>
```

I see that the only difference between the two current Description renderings is the `width: 100%`. Can we remove that? Or can we add it all the time?